### PR TITLE
GUVNOR-2903: [Library] Incorrect navigation when single project is created

### PR DESCRIPTION
**Pull request by paulovmr**

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java [hunk 1]
```diff
@@ -186,13 +188,21 @@ public void showDocks() {
...
-                || place.getIdentifier().equals(EMPTY_LIBRARY_SCREEN)
-                || place.getIdentifier().equals(LIBRARY_SCREEN)
-                || place.getIdentifier().equals(EMPTY_PROJECT_SCREEN)
-                || place.getIdentifier().equals(PROJECT_SCREEN)
-                || place.getIdentifier().equals(PROJECT_DETAIL_SCREEN)
-                || place.getIdentifier().equals(PROJECT_SETTINGS);
+        return getLibraryPlaces().contains(place.getIdentifier());
+    }
+
+    private List<String> getLibraryPlaces() {
```
Comment (jhrcek) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java]: Is it necessary to create new instance of list Strings each time this method is called? Also wouldn't it be better if the list was immutable (`Collections.unmodifiableList`)?

Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java]: Fixed, thanks.

**Comment by jhrcek** [comment]
@paulovmr Thanks, +1

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java [hunk 2]
```diff
@@ -186,13 +200,7 @@ public void showDocks() {
...
 
     private boolean isLibraryPlace(final PlaceRequest place) {
-        return place.getIdentifier().equals(NEW_PROJECT_SCREEN)
-                || place.getIdentifier().equals(EMPTY_LIBRARY_SCREEN)
-                || place.getIdentifier().equals(LIBRARY_SCREEN)
-                || place.getIdentifier().equals(EMPTY_PROJECT_SCREEN)
-                || place.getIdentifier().equals(PROJECT_SCREEN)
-                || place.getIdentifier().equals(PROJECT_DETAIL_SCREEN)
-                || place.getIdentifier().equals(PROJECT_SETTINGS);
+        return LIBRARY_PLACES.contains(place.getIdentifier());
```
Comment (Rikkola) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/util/LibraryPlaces.java]: Nice, I like this

**Pull request merged**